### PR TITLE
Bump christophwurst/id3parser to v0.1.1

### PR DESCRIPTION
**Pull request by ChristophWurst**

### CODE REVIEW

**Comment by rullzer** [comment]
Conflict!

File: composer/installed.json [hunk 1]
```diff
@@ -2094,17 +2094,17 @@
     },
     {
         "name": "pear/archive_tar",
-        "version": "1.4.7",
-        "version_normalized": "1.4.7.0",
+        "version": "1.4.8",
+        "version_normalized": "1.4.8.0",
```
Comment (ChristophWurst) [inline composer/installed.json]: composer wouldn't update my other package without this change :man_shrugging:

Comment (rullzer) [reply composer/installed.json]: sure then :)

**Comment by rullzer** [comment]
Mmmm where is the actual id3 update?

**Pull request merged**